src/main/java/com/noticemedan/map/model/KDTree/Forest.java

```diff
@@ -1,17 +1,27 @@
 package com.noticemedan.map.model.KDTree;
 
 import com.noticemedan.map.model.MapObject;
-import lombok.Getter;
-import lombok.Setter;
-import java.util.List;
+import lombok.AllArgsConstructor;
 
+import java.util.ArrayList;
+import java.util.List;
 
+@AllArgsConstructor
 public class Forest implements ForestInterface {
-	@Setter private KDTree tree;
+	private KDTree trees[];
 
 	@Override
+	public List<MapObject> rangeSearch(Rect query, int zoomLevel) {
```

Comment:
Rect typen og navnet query synes jeg ikke er vildt beskrivende

Reply:
Ved godt Rect ikke bliver defineret her

Reply:
Ja Rect er måske ikke så beskrivende (er kort for Rectangle). Hvad vil du foreslå?
Det skal egentligt lidt forstås som "Efterspørgselsrektangel / søgerektangel / søgeområde". Metoden er tilsvarende: "Jeg returnerer en liste af mapobjekter inden for den rektangel du har efterspurgt".

Reply:
Måske self bare rectangle;)